DeploymentCloud/Deployment.DataX/common.parameters.txt

```diff
@@ -20,6 +20,18 @@ subscriptionId=
 # ResourceGroupName to generate resources 
 resourceGroupName=DataX
 
+# y if you want to run spark jobs on Databricks, n for running jobs on HDinsight 
+useDatabricks=n
+
+#Databricks cluster spark version
+databricksClusterSparkVersion=5.3.x-scala2.11
+
+#Databricks cluster node type id
+databricksClusterNodeType=Standard_DS3_v2
+
+#Databricks SKU
+databricksSku=premium
```

Comment:
should this be premium? How much is this per month?

Reply:
Yes we need premium as it handles the security capabilities. The cost of premium SKU is $0.55/DBU while the cost of standard SKU is $0.40/DBU